docs/options.rst

```diff
@@ -22,7 +22,8 @@ General Options:
 ``JWT_ACCESS_TOKEN_EXPIRES``      How long an access token should live before it expires. This
                                   takes a ``datetime.timedelta``, and defaults to 15 minutes
 ``JWT_REFRESH_TOKEN_EXPIRES``     How long a refresh token should live before it expires. This
-                                  takes a ``datetime.timedelta``, and defaults to 30 days
+                                  takes a ``datetime.timedelta``, and defaults to 30 days.
+                                  Can be set to ``datetime.timedelta(0)`` to disable expiration.
```

Comment:
I wonder if it would be more intuitive to set the expires delta to `False` instead of `datetime.timedelta(0)`. I think that may be more clear about how it is used and the purpose of it, if someone looking at this out in the wild somewhere. 

I think it would only need to be updated in the documentation, because the create_token methods explicitly check for if expires delta is `None`: https://github.com/vimalloc/flask-jwt-extended/blob/master/flask_jwt_extended/jwt_manager.py#L375

Reply:
Yes that might be more intuitive.
JWT_ACCESS_TOKEN_EXPIRES currently must be a datetime.timedelta: https://github.com/vimalloc/flask-jwt-extended/blob/master/flask_jwt_extended/config.py#L154 
So there is some code impact, but that should be minor.